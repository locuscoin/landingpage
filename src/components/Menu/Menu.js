import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

import logo from '../../assets/logo-white-256-v1.0.png';

const Image = styled.img`
  display: block;
  height: 70px;
  width: 70px;
  padding 10px 5px;
`;

const Menu = ({ className }) =>
  <nav className={className}>
    <Image src={logo} />
  </nav>;

const StyledNav = styled(Menu)`
  position: absolute;
  z-index: 2;
`;

export default StyledNav;
