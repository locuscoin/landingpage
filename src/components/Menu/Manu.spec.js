import React from 'react';
import renderer from 'react-test-renderer';

import Menu from './Menu';

describe('Menu component', () => {
  it('should render without crashing', () => {
    const component = renderer.create(<Menu />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
