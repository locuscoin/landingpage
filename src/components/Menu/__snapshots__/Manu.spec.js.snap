// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component should render without crashing 1`] = `
<nav
  className="sc-bwzfXH kndEmM"
>
  <img
    className="sc-bdVaJa sdCSB"
    src="logo-white-256-v1.0.png"
  />
</nav>
`;
